// 処理系のテスト２２
// 異常ケース１１: 返り値の型が宣言と定義で異なる

int move(int x);

void main()
{
  int log = 7;
}

// 返り値の型が異なるのでエラー
void move(int x)
{ 
  int log = 2;
}
